airlock/business_logic.py

```diff
@@ -212,24 +213,19 @@ def abspath(self, relpath):
         The first part of the relpath is the group, so we parse and validate that first.
         """
         relpath = UrlPath(relpath)
-        root = self.root()
-
         group = relpath.parts[0]
+        file_relpath = UrlPath(*relpath.parts[1:])
```

Comment:
Huh, I've been doing `relpath.relative_to(relpath.parts[0])` to do this.

I think this is probably cleaner. Shame there isn't a slice API on Path!

Reply:
I wasn't quite clear whether you were saying your version is cleaner or mine is? Happy with either, as neither is really a thing of beauty.

Reply:
I meant yours. Sliceing the tail of something with `[1:]` is fairly idomatic, and whilst its not quite as simply as that, I think its probably still more readable.